guides/managing.md

```diff
@@ -2,125 +2,164 @@
+all the staff who need to follow them.
 
 ## Playbook governance and workflow
 
 ### Ownership
 
-Each section of the Playbook has an identified owner, typically either the director of the relevant part of the business, or the head of the relevant profession.
+Each section of the Playbook has an identified owner, typically either the
+director of the relevant part of the business, or the head of the relevant
+profession.
 
 The owner is responsible for making sure that their parts of the Playbook:
 
- - continue to reflect our current ways of working and policies
- - are findable, readable, understandable and actionable by its intended audience and users
+- continue to reflect our current ways of working and policies
+- are findable, readable, understandable and actionable by its intended audience
+  and users
 
 The owners of each section are:
 
- - David Mann - [Who we are](https://playbook.dxw.com/#/?id=who-we-are)
- - Adam Maddison - [The work we do > Sales](https://playbook.dxw.com/#/?id=sales)
- - Clare Young - [The work we do > Building services](https://playbook.dxw.com/#/?id=building-services)
- - Dominic Baggott - [The work we do > Hosting and supporting services](https://playbook.dxw.com/#/?id=hosting-and-supporting-services)
- - Wendy Coello - [The work we do > Sharing our expertise](https://playbook.dxw.com/#/?id=sharing-our-expertise)
- - Poss Apostolou - [Working here](https://playbook.dxw.com/#/?id=working-here)
- - Coca Rivas - [Our professions](https://playbook.dxw.com/#/?id=our-professions)
+- David Mann - [Who we are](/#who-we-are)
+- Adam Maddison - [The work we do > Sales](/#sales)
+- Clare Young - [The work we do > Building services](/#building-services)
+- Dominic Baggott -
+  [The work we do > Hosting and supporting services](/#hosting-and-supporting-services)
+- Wendy Coello -
+  [The work we do > Sharing our expertise](/#sharing-our-expertise)
+- Poss Apostolou - [Working here](/#working-here)
+- Coca Rivas - [Our professions](/#our-professions)
 
 ### Workflow
 
-For smaller corrections and updates to the Playbook, staff should consult with relevant colleagues and follow the
+For smaller corrections and updates to the Playbook, staff should consult with
+relevant colleagues and follow the
 [guide to contributing to the Playbook](https://playbook.dxw.com/#/contributing).
 
 For more significant additions and changes to the Playbook, the steps are:
 
- - the owner of the section and the author agree the need for the update
- - the author drafts and reviews the update with appropriate colleagues
- - the author follows the [guide to contributing to the Playbook](https://playbook.dxw.com/#/contributing)
- - the owner and colleagues review and approve the changes
+- the owner of the section and the author agree the need for the update
+- the author drafts and reviews the update with appropriate colleagues
+- the author follows the
+  [guide to contributing to the Playbook](https://playbook.dxw.com/#/contributing)
```

Comment:
Should this become a `/guides/` link instead of using the octothorp? ( ;) )

Reply:
Yup. I caught that too :).